### PR TITLE
tuir: add page

### PR DESCRIPTION
**Pull request by GianniBYoung**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/tuir.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# tuir
+
+> A text-based interface (TUI) to view and interact with Reddit from your terminal.
+> Navigate with the Vim keys. More information: https://gitlab.com/ajak/tuir.
```
Comment (navarroaxel) [inline pages/common/tuir.md:4]:
```suggestion
> Navigate with the Vim keys.
> More information: <https://gitlab.com/ajak/tuir>.
```

**Review by bl-ue — APPROVED** [review]

**Review by schneiderl — APPROVED** [review]
Did not know about this tool. Seems nice :)

File: pages/common/tuir.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# tuir
+
+> A text-based interface (TUI) to view and interact with Reddit from your terminal.
```
Comment (bl-ue) [inline pages/common/tuir.md:3]: TUI doesn't stand for "text-based interface" :wink:
```suggestion
> A text user-interface (TUI) to view and interact with Reddit from your terminal.
```

**Review by sbrl — APPROVED** [review]
Thanks, @GianniBYoung!

**Pull request merged**